package.json

```diff
@@ -24,7 +24,7 @@
     "version": "auto-changelog -p --config .auto-changelog.json --unreleased && git add CHANGELOG.md"
   },
   "devDependencies": {
-    "@azure/functions": "^1.2.2",
+    "@azure/functions": "^3.2.0",
```

Comment:
should we also update `durable-functions` dependency? we are currently using a much outdated version.
I also see that `azure-storage` is now deprecated. To be more consistent with the runtime update I think we should update those dependencies.

Reply:
Yes, I think we must update also those, and be very careful! Anyway I'm not seeing any durable function in this repo, I'd check if the import is really needed.